src/editor.rs

```diff
@@ -653,13 +653,13 @@ impl<'a> Editor<'a> {
         for _ in 0..num_rows {
             current = (current + if forward { 1 } else { num_rows - 1 }) % num_rows;
             let row = &mut self.rows[current];
-            if let Some(rx) = row.render.find(&query) {
+            if let Some(cx) = row.render.find(&query) {
```

Comment:
`str::find` returns byte index instead of character index.

Reply:
`cx` usually represents the byte index in `row.chars`, not `row.render`.